pkg/inventory/inventory-client_test.go

```diff
@@ -180,7 +180,7 @@ func TestMerge(t *testing.T) {
 				if !tc.isError && err != nil {
 					t.Fatalf("unexpected error: %s", err)
 				}
-				if !object.SetEquals(tc.pruneObjs, pruneObjs) {
+				if !object.IsSetEqual(tc.pruneObjs, pruneObjs) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.